doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -154,6 +138,58 @@ each replica, including the times at which it was created and dropped
 | `dropped_at`          | [`timestamp with time zone`] | The time at which the replica was dropped, or `NULL` if it still exists.                                                                  |
 | `credits_per_hour`    | [`numeric`]                  | The number of compute credits consumed per hour. Corresponds to [`mz_cluster_replica_sizes.credits_per_hour`](#mz_cluster_replica_sizes). |
 
+### `mz_frontiers`
+
+The `mz_frontiers` table describes the frontiers of each source, sink, table,
+materialized view, index, and subscription in the system, as observed from the
+coordinator.
+
+For objects that are installed on replicas (e.g., materialized views and
+indexes), the `replica_id` field is always non-`NULL`. If an object is installed
+on multiple replicas, it has multiple entries describing the frontier on each
+individual replica. For objects that are not installed on replicas (e.g.,
+tables), the `replica_id` field is `NULL`.
+
+If you don't care about per-replica frontiers, using the
+[`mz_global_frontiers`](#mz_global_frontiers) view instead is more convenient.
+
+[`mz_compute_frontiers`](#mz_compute_frontiers) is similar to this table, but
+`mz_compute_frontiers` reports the frontiers known to the active compute
+replica, while this table reports the frontiers of all replicas. Note also that
```

Comment:
This is saying that `mz_compute_frontiers` is similar to the `mz_global_frontiers` table?

```suggestion
[`mz_compute_frontiers`](#mz_compute_frontiers) is similar to the `mz_global_frontiers` table, but
`mz_compute_frontiers` reports the frontiers known to the active compute
replica, while the `mz_global_frontiers` table reports the frontiers of all replicas. Note also that
```

Reply:
Oh no, "this table" is supposed to refer to the table this piece of the docs belong to, i.e. `mz_frontiers`.